client/js/widgets/widget.js

```diff
@@ -599,6 +601,33 @@ export class Widget extends StateManaged {
           }
         }
       }
+      if(a.func == 'IF') {
+        setDefaults(a, { relation: '==' });
+        if (!['==', '!=', '<', '<=', '>=', '>'].indexOf(a.relation) < 0) {
```

Comment:
Did you mean to remove the `!`? If you didn't, I really don't understand how this works. :D

Reply:
yeah, thanks